selfdrive/car/volkswagen/values.py

```diff
@@ -153,6 +155,7 @@ class CAR:
       b'\xf1\x875Q0959655BH\xf1\x890336\xf1\x82\02314160011123300314211012230229333463100',
       b'\xf1\x875Q0959655BT\xf1\x890403\xf1\x82\023141600111233003142405A2252229333463100',
       b'\xf1\x875Q0959655J \xf1\x890830\xf1\x82\023271112111312--071104171825102591131211',
+      b'\xf1\x875Q0959655J \xf1\x890830\xf1\x82\023272512111312--07110417182C102C91131211',
```

Comment:
```suggestion
```
CI kicked this one back because it's a duplicate of a FW entry a few lines down. If you remove it, CI should pass and it can be merged.